docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -245,6 +246,10 @@ t.track_page_ping("http://mytesturl/test2", "Page title 2", "http://myreferrer.c
 
 ### Track screen view
 
+:::note
```

Comment:
Also would be nice to link the schema here (maybe a link to Github) and also add `(deprecated)` to the section title to make it more clear.

Reply:
Actually, maybe it would be better to just mention this deprecated screen view API as a note under the `Track mobile screen view` section (that could also be renamed to just `Track screen view`)? A bit worried about confusing people with having two similar sections and we want to direct people to the new API.